src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -1077,21 +1077,18 @@ private static string NormalizeDotnetFrameworkVersion(string version)
 
             if (version == null)
             {
-                // Legacy behavior.
-                parsedVersion = new Version("5.0");
+                parsedVersion = new Version(_requiredNetFrameworkVersion);
             }
             else if (!Version.TryParse(version, out parsedVersion))
             {
                 // remove any leading "v" and try again
                 if (!Version.TryParse(version.ToLower().TrimStart('v'), out parsedVersion))
                 {
-                    throw new CliException($"The dotnet-version value of '{version}' is invalid. Specify a value like '5.0'.");
+                    throw new CliException($"The dotnet-version value of '{version}' is invalid. Specify a value like '{_requiredNetFrameworkVersion}'.");
                 }
             }
 
-            string normalizedVersion = $"{parsedVersion.Major}.{parsedVersion.Minor}";
-            ColoredConsole.WriteLine($"Using dotnet-version value of '{normalizedVersion}'.");
```

Comment:
Decided there wasn't really a need to write this out during parsing as it can be noisy; we'll still log if we change it.